uat/.gitignore

```diff
@@ -0,0 +1,12 @@
+target
```

Comment:
We don't need this .gitignore file here. Add this exclusions to the too `.gitignore`

Reply:
removed